src/Store/ExtensionClient.rei

```diff
@@ -0,0 +1,7 @@
+open Oni_Core;
+open Oni_Model;
+open Oni_Extensions;
+
+let create:
+  (~extensions: list(ExtensionScanner.t), ~setup: Setup.t) =>
+  (ExtHostClient.t, Isolinear.Stream.t(Actions.t));
```

Comment:
I split the ExtHostClient creation out into a separate module, so it can be instantiated by `StoreThread` and be used outside the `ExtensionClientStore`.

Reply:
Cool, that sounds good.